windeploy.sh

```diff
@@ -3,7 +3,7 @@
 LOG=windeploy.log
 
 # Detecting current pgModeler version
-DEPLOY_VER=`cat libutils/src/globalattributes.cpp | grep PgModelerVersion | sed 's/PgModelerVersion=QString("//g' | sed 's/")//g'`
+DEPLOY_VER=`cat libutils/src/globalattributes.cpp | grep PgModelerVersion | sed 's/PgModelerVersion(QString("//g' | sed 's/")//g'`

```

Comment:
?

Reply:
Now it compiles, and run, still don't really understand the purpose of this, and how the fallback dir changes.

Reply:
`windeploy.sh` prints the current version of pgModeler on the shell and gets that value from `globalattributes.cpp` With switching to a class, that version number simply needs to be read slightly different.